assets/espresso/pw_bands_multimaterial.j2.in

```diff
@@ -0,0 +1,44 @@
+{% set MATERIAL_INDEX = subworkflowContext.MATERIAL_INDEX -%}
```

Comment:
Maybe we can just update the template as:

```jinja
set input = input.perMaterial[MATERIAL_INDEX] if subworkflowContext.MATERIAL_INDEX else input
```
```

Reply:
Ok. It will have to look a bit differently though. I just checked and an assignment with inline if/else does not work. The workaround would be:
```jinja
{% if subworkflowContext.MATERIAL_INDEX %}
{%- set input = input.perMaterial[MATERIAL_INDEX] -%}
{% endif -%}
```